docker-compose.yml

```
@@ -18,3 +18,17 @@ services:
     environment:
       ADMINER_PLUGINS: 'tables-filter'
       ADMINER_DEFAULT_SERVER: 'postgres'
+
+  mysql:
+    image: mysql:5
+    environment:
+      MYSQL_ROOT_PASSWORD: 'secret'
+      MYSQL_DATABASE: 'adp-dev'
+      MYSQL_USER: 'adp'
+      MYSQL_PASSWORD: 'secret'
+    command: --init-file /data/application/init.sql
+    volumes:
+      - ./mysql:/var/lib/mysql
```

Could you put this under `var` directory, and add `/var/` to `.gitignore`. We already have `postgres` directory, it would be nice to move it to `var` directory too. This way, we will have all service data in one place.

```suggestion
      - ./var/mysql:/var/lib/mysql
```